pos-print/src/main/java/com/clouway/pos/print/adapter/db/PersistentCashRegisterRepository.kt

```diff
@@ -56,19 +56,28 @@ class PersistentCashRegisterRepository @Inject constructor(private val database:
   }
 
   private fun adapt(record: Document): CashRegister {
+    val policy = record["fiscalPolicy"] as Document
```

Comment:
Make check here for existence as there are persisted configurations in production which will 
require manual reconfiguration if we don't make it.

```
if doesn't have policy 
   then provide empty set of policies
```

Reply:
fixed